skiko/src/jsMain/kotlin/org/jetbrains/skiko/SkiaLayer.js.kt

```diff
@@ -66,36 +49,26 @@ actual open class SkiaLayer {
         // https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events
         htmlCanvas.addEventListener("pointerdown", { event ->
             event as MouseEvent
-            skikoView?.onPointerEvent(translatePointerEvent(
-                event, true, SkikoPointerEventKind.DOWN))
+            skikoView?.onPointerEvent(toSkikoEvent(event, true, SkikoPointerEventKind.DOWN))
         })
         htmlCanvas.addEventListener("pointerup", { event ->
             event as MouseEvent
-            var mask = SkikoMouseButtons.NONE
-            skikoView?.onPointerEvent(translatePointerEvent(
-                event, true, SkikoPointerEventKind.UP))
+            skikoView?.onPointerEvent(toSkikoEvent(event, true, SkikoPointerEventKind.UP))
         })
         htmlCanvas.addEventListener("pointermove", { event ->
             event as MouseEvent
-            skikoView?.onPointerEvent(translatePointerEvent(
-                event, false, SkikoPointerEventKind.MOVE))
+            skikoView?.onPointerEvent(toSkikoEvent(event, false, SkikoPointerEventKind.MOVE))
```

Comment:
```
toSkikoEvent(event, false, SkikoPointerEventKind.MOVE)
```
as I understand `false` here means that there are no buttons pressed.

But it's possible that pointer can be moved while some buttons are pressed, so users will receive skiko move event without any information about pressed buttons. Do I understand current logic correctly?

I see `false` was there before these changes, so I'm wondering if `false` is intentional?